### PR TITLE
Write "looking ahead" sections for all chapters

### PR DESCRIPTION
**Pull request by bbdaniels**

### CODE REVIEW

File: chapters/4-acquisition.tex [hunk 1]
```diff
@@ -726,15 +726,25 @@ \subsection{Backing up original data}
...
+giving you a range of tools and a complete workflow
+to transferring and storing data securely at all times.
+Once your original data is completely transferred, securely stored, and backed up,
+the data acquisition stage is complete, as
+summarized in the figure accompanying this chapter.
+This is when the heavy lifting with statistical software starts.
+Before you can proceed to analyze your data 
+and answer your project's research questions,
+you first need to check the quality of the data acquired
 and make sure you have the right information in the right format.
```
Comment (mariaruth) [inline chapters/4-acquisition.tex:746]:
```suggestion
you first need to check the quality of the data acquired,
make sure you have the right information in the right format, and prepare it for analysis.
```

File: chapters/5-processing.tex [hunk 2]
```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
 so other researchers may use them for their analysis as well.
 
 \section{Looking ahead}
-Once you have corrected, cleaned, 
-and de-identified one or several data tables,
-and compiled the documentation to support it,
+This chapter introduced a process of exploring, understanding, and correcting
```
Comment (mariaruth) [inline chapters/5-processing.tex:831]:
```suggestion
This chapter introduced a process of exploring, understanding, and processing
```

File: chapters/5-processing.tex [hunk 3]
```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
...
+the data that you obtained from the field or from partners.
+This process, called data cleaning and processing,
+creates your first research outputs using original data:
```
Comment (mariaruth) [inline chapters/5-processing.tex:834]:
```suggestion
This process creates your first research output using original data:
```

File: chapters/5-processing.tex [hunk 4]
```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
...
+an analysis-ready dataset.
+This dataset is a clear representation of reality (it is ``tidy''),
```
Comment (mariaruth) [inline chapters/5-processing.tex:836]: this implied definition of tidy doesn't seem right to me

File: chapters/5-processing.tex [hunk 5]
```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
...
+it faithfully represents the measurements it was intended to collect,
+and it does not expose the identities of the people described by it.
+You have also now taken the time to fully understand the patterns and structures
+in your data, and annotated and labeled them for your use and use by others.
+Once you have completed these steps and
+and compiled the documentation to accompany your work,
 you have created the first research output of your project: an original dataset.
```
Comment (mariaruth) [inline chapters/5-processing.tex:843]: this repeats the line above (about creating the first research output). i suggest removing all of this, unless we want to add a separate point about documentation that is necessary to make this a useful output for the research process going forward

File: chapters/5-processing.tex [hunk 6]
```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
...
-This will help you organize your files and create a backup of the data,
-and some donors require that the data be filed as an intermediate step of the project.
-However, the reason why you acquired and cleaned this data
-was a research question.
+Combined with the data map, this product
```
Comment (mariaruth) [inline chapters/5-processing.tex:844]:
```suggestion
Combined with the data map, this dataset
```

File: chapters/5-processing.tex [hunk 7]
```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
...
+and compiled the documentation to accompany your work,
 you have created the first research output of your project: an original dataset.
-This will help you organize your files and create a backup of the data,
-and some donors require that the data be filed as an intermediate step of the project.
-However, the reason why you acquired and cleaned this data
-was a research question.
+Combined with the data map, this product 
+will be the fundamental starting point for all analysis work.
+Archiving, cataloging, and publishing this dataset 
+will organize your files and create a backup of the data.
```
Comment (mariaruth) [inline chapters/5-processing.tex:847]: i don't understand this sentence

File: chapters/5-processing.tex [hunk 8]
```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
...
 Though data cleaning is typically the most time-consuming process in the data work,
-the reason why it is done is preparing data to be analyzed.
-In Chapter 6, you will see what are the last few steps needed to finally run the
-regressions originally specified in your analysis plan
+the reason why it is done is preparing data to be analyzed
+to answer a set of research questions.
```
Comment (mariaruth) [inline chapters/5-processing.tex:850]: this sentence is also a bit awkward. do we need it? not sure of the point it's trying to make. maybe something like, although the process described in this chapter is typically very time-consuming, investing in a careful research product at this point will simplify all workflows going forward

File: chapters/5-processing.tex [hunk 9]
```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
...
+In the next chapter, we will walk through the steps needed 
+to finally run the analyses originally specified in your research plan
```
Comment (mariaruth) [inline chapters/5-processing.tex:852]:
```suggestion
to actually run the analyses originally specified in your pre-analysis plan
```

File: chapters/6-analysis.tex [hunk 10]
```diff
@@ -745,6 +745,29 @@ \subsection{Using {\LaTeX} for dynamic research outputs}
 so papers can be more easily be formatted into their specific layout.
 
 \section{Looking ahead}
+This section discussed the steps needed to take original data
+and create analysis datasets and outputs from it.
```
Comment (mariaruth) [inline chapters/6-analysis.tex:749]:
```suggestion
This chapter discussed the steps needed to 
create analysis datasets and outputs from original data.
```

File: chapters/6-analysis.tex [hunk 11]
```diff
@@ -745,6 +745,29 @@ \subsection{Using {\LaTeX} for dynamic research outputs}
...
+This process created new derived information
+from measurements taken in the field or collected by partners,
+and combined these measures with the research design
+described in the data map and the analysis plan.
```
Comment (mariaruth) [inline chapters/6-analysis.tex:753]: how can information be 'combined' with the research design. perharps organize it for analysis according to the research design, as outlined in the Data Map?

File: chapters/6-analysis.tex [hunk 12]
```diff
@@ -745,6 +745,29 @@ \subsection{Using {\LaTeX} for dynamic research outputs}
...
+without access to detailed records and explanations
+of how you interpreted and modified the data.
+Creating organized and well-documented code
+for both the creation of new information
+and the creation of research outputs describing
+patterns you identified in the data --
+and used to answer your research questions --
+provides this knowledge to both future users of the original data
+as well as readers interested in completely understanding
+the results you obtained.
```
Comment (mariaruth) [inline chapters/6-analysis.tex:765]: Overly complex sentence. perhaps:
You learned that code must be well-organized and well-documented to allow others to understand how research outputs were created and how those were used to answer the research questions. 
?

File: chapters/6-analysis.tex [hunk 13]
```diff
@@ -745,6 +745,29 @@ \subsection{Using {\LaTeX} for dynamic research outputs}
...
+The final chapter of this book will provide a guide
+to pulling these raw findings together into publishable work,
```
Comment (mariaruth) [inline chapters/6-analysis.tex:767]:
```suggestion
to assembling the raw findings into publishable work,
```

File: chapters/7-publication.tex [hunk 14]
```diff
@@ -637,6 +637,22 @@ \subsection{Releasing a reproducibility package}
 as their access is more restrictive,
 and organizations limit access to platforms other than the one officially adopted.
 
+\section{Looking ahead}
+This chapter described a the process that brings all your efforts together --
```
Comment (mariaruth) [inline chapters/7-publication.tex:641]:
```suggestion
This chapter described the culmination of all the efforts in data acquisition, cleaning, processing, and analyzing --
```

File: chapters/7-publication.tex [hunk 15]
```diff
@@ -637,6 +637,22 @@ \subsection{Releasing a reproducibility package}
...
+the production of materials to share with the world
+that combine design, data, processing, and analysis
+to answer a scientific or policy question in a way that has
```
Comment (mariaruth) [inline chapters/7-publication.tex:644]:
```suggestion
that answer a scientific or policy question in a way that has 
```

File: chapters/7-publication.tex [hunk 16]
```diff
@@ -637,6 +637,22 @@ \subsection{Releasing a reproducibility package}
...
 
+\section{Looking ahead}
+This chapter described a the process that brings all your efforts together --
+the production of materials to share with the world
+that combine design, data, processing, and analysis
+to answer a scientific or policy question in a way that has 
+never been done before.
+Making sure that everything you discovered and created
+is as broadly available and as easy to use as possible
+is the last step in producing scientific work from original data.
```
Comment (mariaruth) [inline chapters/7-publication.tex:648]:
```suggestion
is the last step in producing scientific evidence from original data.
```

File: chapters/7-publication.tex [hunk 17]
```diff
@@ -637,6 +637,22 @@ \subsection{Releasing a reproducibility package}
...
+This is the purpose of all the rigor, organization, and documentation
+we encourage and detail at every step of the process:
+since all your research materials are 
+continuously organized, shareable, secure, documented, and readable,
+they are both valuable to you and others and easy to provide access to.
+From here on, what to do with that is your decision.
```
Comment (mariaruth) [inline chapters/7-publication.tex:654]: This seems like a strange last word ... i would just leave it with the previous sentence

**Review by mariaruth — CHANGES_REQUESTED** [review]
Lots of revisions suggested in line. You referenced the diagram in one of the sections and i think it would be really useful to do in all.

File: chapters/2-collaboration.tex [hunk 18]
```diff
@@ -737,3 +737,22 @@ \subsection{Ensuring privacy and security in research data}
 All data should have an expiration data and after a project is completed and
 de-identified data has been made public as a part of your publication,
 there is no need for research teams to sit on confidential data indefinitely.
+
+\section{Looking ahead}
+With your code environment established,
+you will have a firm idea about how we are going to
```
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/2-collaboration.tex:743]:
```suggestion
With your code environment established,
you will have a firm idea about how you are going to
```

Yep! I started writing with "we" and then changed it, missed a few I'm sure!

File: chapters/2-collaboration.tex [hunk 19]
```diff
@@ -737,3 +737,22 @@ \subsection{Ensuring privacy and security in research data}
 All data should have an expiration data and after a project is completed and
 de-identified data has been made public as a part of your publication,
 there is no need for research teams to sit on confidential data indefinitely.
```
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/2-collaboration.tex:739]:
```suggestion
All data should have a clear data retention and destruction plan.
After a project is completed and its
de-identified data has been made available as a part of data publication,
research teams should not retain confidential data indefinitely.
```

File: chapters/2-collaboration.tex [hunk 20]
```diff
@@ -737,3 +737,22 @@ \subsection{Ensuring privacy and security in research data}
...
+With an organization plan
+and plans to to version-control and back up files,
+you are also ready to handle materials ethically and securely.
+You should also have secured the approvals needed for any planned work.
+You are now ready to start creating the data materials needed
+to answer your research questions.
+In the next chapter,
+we will do this through an essential type of research planning:
+mapping data to both the research design
+and to its planned creation and use across the project timeline.
```
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/2-collaboration.tex:758]:
```suggestion
we will outline how to prepare the essential elements of research data.
You will learn how to map out your project's data needs
according to both the research design
and the planned creation and use of data across the project timeline.
```

File: chapters/3-measurement.tex [hunk 21]
```diff
@@ -931,3 +931,23 @@ \section{Doing power calculations for research design}
 or if results seem to depend dramatically on the outcomes for a small number of individuals,
 randomization inference will flag those issues before the experiment is fielded
 and allow adjustments to the design to be made.
+
+\section{Looking ahead}
+This chapter provided a framework for you
+plan the data work that corresponds to the design of your experiment.
+These research variables and the data map you create
+to catalog, link, and describe them
```
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/3-measurement.tex:939]: In this one I want to leave the "research variables" reference explicitly, to show that this is data work _using_ the design information (the data map). Is there a clearer way to express that?

File: chapters/3-measurement.tex [hunk 22]
```diff
@@ -931,3 +931,23 @@ \section{Doing power calculations for research design}
 or if results seem to depend dramatically on the outcomes for a small number of individuals,
 randomization inference will flag those issues before the experiment is fielded
 and allow adjustments to the design to be made.
+
+\section{Looking ahead}
+This chapter provided a framework for you
+plan the data work that corresponds to the design of your experiment.
```
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/3-measurement.tex:937]: In this one I want to set up the "research variables" reference explicitly (in the next comment), to show that this is data work using the design information (the data map). Is there a clearer way to express that?

File: chapters/5-processing.tex (hunk 2 above)
Comment (bbdaniels) [reply chapters/5-processing.tex:831]: That is redundant with "this process ... is called processing" on Line 833. Which is also poorly written... Far too much "processing" going on here. Do we have other terms available for this?

File: chapters/5-processing.tex (hunk 3 above)
Comment (bbdaniels) [reply chapters/5-processing.tex:834]:
```suggestion
These tasks create your first research outputs using original data:
```

File: chapters/5-processing.tex (hunk 2 above)
Comment (bbdaniels) [reply chapters/5-processing.tex:831]:
```suggestion
This chapter introduced a workflow of formatting, cleaning, and correcting
```

Do you not like "correcting" specifically?

File: chapters/5-processing.tex (hunk 4 above)
Comment (bbdaniels) [reply chapters/5-processing.tex:836]:
```suggestion
This dataset is well-structured to describe your units of analysis (it is ``tidy''),
```

File: chapters/5-processing.tex (hunk 5 above)
Comment (bbdaniels) [reply chapters/5-processing.tex:843]:
```suggestion
You have also now taken the time to fully understand the patterns and structures
in your data, and annotated and labeled them for your use and use by others.
```

File: chapters/5-processing.tex (hunk 7 above)
Comment (bbdaniels) [reply chapters/5-processing.tex:847]:
```suggestion
```

File: chapters/5-processing.tex (hunk 9 above)
Comment (bbdaniels) [reply chapters/5-processing.tex:852]:
```suggestion
to run the analyses originally specified in your analysis plan
```

File: chapters/2-collaboration.tex (hunk 19 above)
[the remark this answers is not in the document]
Comment (mariaruth) [reply chapters/2-collaboration.tex:739]: How about, 'Every project should have a clear data retention and destruction plan.

File: chapters/6-analysis.tex (hunk 11 above)
Comment (bbdaniels) [reply chapters/6-analysis.tex:753]: Same as above -- this is meant to refer to merging with master datasets according to the other plans, so literally to be merging research variables and measurement variables as we've defined. How to better express that?

File: chapters/6-analysis.tex (hunk 12 above)
Comment (bbdaniels) [reply chapters/6-analysis.tex:765]:
```suggestion
You learned that code must be well-organized and well-documented 
to allow others to understand how research outputs were created 
and how those were used to answer the research questions.
```

File: chapters/7-publication.tex (hunk 17 above)
Comment (bbdaniels) [reply chapters/7-publication.tex:654]:
```suggestion
```

File: chapters/2-collaboration.tex (hunk 19 above)
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/2-collaboration.tex:739]: As in, delete the rest?

File: chapters/3-measurement.tex (hunk 21 above)
[the remark this answers is not in the document]
Comment (mariaruth) [reply chapters/3-measurement.tex:939]: Ok. the way this is phrased makes it seem like the data map is cataloging/linking/describing the research variables, that's what i didn't like. not very sure how to clarify. Maybe something like: You learned how to translate the research design into specific research variables, and created a data map to guide construction of the analysis dataset.

File: chapters/2-collaboration.tex [hunk 23]
```diff
@@ -737,3 +737,22 @@ \subsection{Ensuring privacy and security in research data}
...
+handle the data and code that
+you create and receive throughout the research process.
+This structure should prepare you to work collaboratively,
+to share code and data across machines and among team members,
+and to document your work as a group.
+With an organization plan
+and plans to to version-control and back up files,
+you are also ready to handle materials ethically and securely.
+You should also have secured the approvals needed for any planned work.
+You are now ready to start creating the data materials needed
```
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/2-collaboration.tex:753]: Same as below, this is supposed to refer to data map. I didn't want to use the term before it is introduced. Maybe "organizing information about the research design"? But that's clunky.

File: chapters/5-processing.tex (hunk 2 above)
Comment (mariaruth) [reply chapters/5-processing.tex:831]: workflow is a good replacement for processing here. correction raises red flags for me, we don't mean it this way, but too often it's 'correcting' the data to make it look more like what you expected ... that's why i wanted to replace.

File: chapters/2-collaboration.tex (hunk 19 above)
[the remark this answers is not in the document]
Comment (mariaruth) [reply chapters/2-collaboration.tex:739]: nope just revise the first sentence :)

File: chapters/2-collaboration.tex (hunk 23 above)
[the remark this answers is not in the document]
Comment (mariaruth) [reply chapters/2-collaboration.tex:753]: ah, i see. what about 'ready to start translating your project's research design into a clear measurement framework'?

File: chapters/3-measurement.tex (hunk 22 above)
[the remark this answers is not in the document]
Comment (mariaruth) [reply chapters/3-measurement.tex:937]: i think that can be addressed in the next sentence, the phrasing here is so clunky ...

File: chapters/5-processing.tex (hunk 2 above)
Comment (bbdaniels) [reply chapters/5-processing.tex:831]: we call it correction throughout the chapter though, should we review that?

Comment (mariaruth) [reply chapters/5-processing.tex:831]: in context it's ok . i guess fine here too, just makes me nervous! perhaps something like correcting field errors to help limit the scope?

File: chapters/2-collaboration.tex (hunk 19 above)
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/2-collaboration.tex:739]: yep yep yep, edited above

File: chapters/6-analysis.tex (hunk 11 above)
Comment (mariaruth) [reply chapters/6-analysis.tex:753]: How about something along the lines of: By combining the variables of interest for the analysis (measurement variables) with the variables describing the analytical design (research variables), you created original datasets ready for analysis.

File: chapters/2-collaboration.tex (hunk 23 above)
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/2-collaboration.tex:754]:
```suggestion
ready to translate your project's research design into a measurement framework
```

File: chapters/3-measurement.tex (hunk 22 above)
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/3-measurement.tex:937]:
```suggestion
develop a data map that documents the design of your study.
```

File: chapters/3-measurement.tex (hunk 21 above)
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/3-measurement.tex:939]:
```suggestion
The data map contains research variables and the instructions
for using them in combination with measurement variables, which together
```

File: chapters/6-analysis.tex (hunk 11 above)
Comment (bbdaniels) [reply chapters/6-analysis.tex:753]:
```suggestion
By combining the observed variables of interest for the analysis (measurement variables) 
with the information in the data map describing the study design (research variables), 
you created original datasets ready for analysis.
```

File: chapters/3-measurement.tex [hunk 24]
```diff
@@ -931,3 +931,23 @@ \section{Doing power calculations for research design}
...
+These research variables and the data map you create
+to catalog, link, and describe them
+will form the foundation of your analytical work
+when you obtain measurements from your data.
```
Comment (bbdaniels) [inline chapters/3-measurement.tex:941]:
```suggestion
will form the dataset for your analytical work.
```

File: chapters/5-processing.tex (hunk 8 above)
Comment (bbdaniels) [reply chapters/5-processing.tex]: b7bbbdb330e6b9c6c35df47ec4d84e7a88992833

File: chapters/5-processing.tex (hunk 2 above)
Comment (bbdaniels) [reply chapters/5-processing.tex:831]: how about QA as now suggested.

File: chapters/4-acquisition.tex [hunk 25]
```diff
@@ -726,15 +726,25 @@ \subsection{Backing up original data}
...
+and how to make those materials available in the future.
+It also provided an extensive guide for one of the most popular --
+and challenging --
+methods of data collection: primary electronic surveys.
+Finally, it emphasized the secure handling of this potentially sensitive information,
+giving you a range of tools and a complete workflow
+to transferring and storing data securely at all times.
+Once your original data is completely transferred, securely stored, and backed up,
+the data acquisition stage is complete, as
+summarized in the figure accompanying this chapter.
```
[the remark this answers is not in the document]
Comment (bbdaniels) [reply chapters/4-acquisition.tex:741]: 19519d05dabe0ee0527bbe05ce5c32556f931280

File: chapters/3-measurement.tex [hunk 26]
```diff
@@ -931,3 +931,22 @@ \section{Doing power calculations for research design}
 or if results seem to depend dramatically on the outcomes for a small number of individuals,
 randomization inference will flag those issues before the experiment is fielded
 and allow adjustments to the design to be made.
+
+\section{Looking ahead}
+This chapter provided a framework for you
```
Comment (kbjarkefur) [inline chapters/3-measurement.tex:936]:
```suggestion
This chapter provided a framework for you to
```

File: chapters/3-measurement.tex [hunk 27]
```diff
@@ -931,3 +931,22 @@ \section{Doing power calculations for research design}
...
+develop a data map that documents the design of your study.
```
Comment (kbjarkefur) [inline chapters/3-measurement.tex:937]:
```suggestion
develop a data map that documents your data needs and 
how they relate to the design of your study.
```
Something like this is needed. it is not the design that it documents...

File: chapters/4-acquisition.tex [hunk 28]
```diff
@@ -726,15 +726,25 @@ \subsection{Backing up original data}
...
-the data acquisition stage, 
-summarized in the figure accompanying this chapter, is complete.
-This is when the heavy lifting on statistical software starts.
-If you think that now you can proceed to analyze your data and answer your project's research questions,
-think again.
-Before you can do that, you need to check the quality of the data acquired
-and make sure you have the right information in the right format.
-This proces, which we call data cleaning and processing,
+This chapter provided a road map to the acquisition of original data.
+It outlined guidelines for ensuring you have effective ownership
```
Comment (kbjarkefur) [inline chapters/4-acquisition.tex:730]:
```suggestion
It outlined guidelines for ensuring that you have effective ownership
```
You guys tell me, but there seems to be a missing word here

File: chapters/4-acquisition.tex [hunk 29]
```diff
@@ -726,15 +726,25 @@ \subsection{Backing up original data}
...
+or licensing of data that you obtain or collect,
+and how to make those materials available in the future.
+It also provided an extensive guide for one of the most popular --
```
Comment (kbjarkefur) [inline chapters/4-acquisition.tex:733]:
```suggestion
It also provided an extensive guide for one of the most common --
```
Common is more objective that popular.

File: chapters/3-measurement.tex (hunk 27 above)
Comment (bbdaniels) [reply chapters/3-measurement.tex:937]: Hmm, I don't agree, I think the point of master datasets and research variables _is_ to document the study design! or at least its execution.

**Review by mariaruth — APPROVED** [review]

File: chapters/3-measurement.tex (hunk 27 above)
Comment (mariaruth) [reply chapters/3-measurement.tex:937]: Softly in favor of Kris's suggestion, i do think of these as providing a framework for linking all the data to the research questions

Comment (bbdaniels) [reply chapters/3-measurement.tex:937]: Ok -- edited and adopted

**Pull request merged**